test/jit/test_tracer.py

```diff
@@ -1188,6 +1188,17 @@ def tensor_size(x: torch.Tensor) -> torch.Tensor:
             torch.tensor([15])
         )
 
+        @torch.jit.script
+        def use_device(x):
+            return torch.zeros_like(x, device=x.device)
```

Comment:
zeros_like will default the device to the input, maybe use `zero` to test instead?